storage-model/src/main/java/no/unit/nva/publication/storage/model/daos/JoinWithResource.java

```diff
@@ -40,7 +40,7 @@ default String getByResourcePartitionKey() {
     @JsonProperty(BY_RESOURCE_INDEX_SORT_KEY_NAME)
     default String getByResourceSortKey() {
         return
-            this.getType()
+            this.getOrderedType()
```

Comment:
orderedType is "a:DoiRequest" and "b:Resource" so that we enforce an ordering in the "ByResource" index.